index.html

```diff
@@ -75,7 +48,59 @@ <h1>Active Flights</h1>
           ></path>
         </svg>
       </div>
+      <div id="loading-message" class=" bg-primary-bg-color text-center"></div>
+    </div>
+    <div id="container" class="flex h-full flex-col content-start">
+      <header class="heading m-0 flex h-fit bg-primary-bg-color p-0">
+        <div class="logo h-44 w-44"></div>
```

Comment:
why this random div with nothing in it? Does logo have an svg or something?

Reply:
The div I have as the placeholder for where my loading icon will be placed over after loading.